test/ConfigParallelismTestCases.longtest.js

```diff
@@ -0,0 +1,6 @@
+const { describeCases } = require("./ConfigTestCases.template");
+
+describeCases({
+	name: "ConfigParallelismTestCases",
+	parallelism: 1
+});
```

Comment:
These test suite doesn't fail without the changes...

We need some kind of test cases for the bug